CHANGELOG.md

```diff
@@ -33,6 +33,11 @@
 - [#174](https://github.com/SuperGoodSoft/solidus_taxjar/pull/174) Provide a link to Taxjar state settings
 - [#177](https://github.com/SuperGoodSoft/solidus_taxjar/pull/177) Make nexus caching configurable
 - [#169](https://github.com/SuperGoodSoft/solidus_taxjar/pull/169) Add basic backfill transaction functionality
+- [#182](https://github.com/SuperGoodSoft/solidus_taxjar/pull/182) Automatically create default Tax Rate
+
+## Upgrading Instructions
+
+* If you had previously configured a `Spree::TaxRate` with the name "Sales Tax", it can be deleted after upgrading. [#182](https://github.com/SuperGoodSoft/solidus_taxjar/pull/182)
```

Comment:
Could we also suggest the option of renaming the "Sales Tax" calculator to "Solidus TaxJar Rate" if users don't want to end up with a _new_ calculator after this upgrade. 
I know on the project I work we have specs where we explicitly setup this tax rate when we want to get actual taxes through TaxJar which will also have to be updated.

Reply:
I like that idea!

Reply:
Updated!